var/spack/repos/spack_repo/builtin/packages/libxslt/package.py

```diff
@@ -41,22 +45,42 @@ class Libxslt(AutotoolsPackage):
     depends_on("c", type="build")
 
     depends_on("pkgconfig@0.9.0:", type="build")
+
     depends_on("iconv")
     depends_on("libxml2")
     depends_on("libxml2+python", when="+python")
     depends_on("xz")
     depends_on("zlib-api")
+
     depends_on("libgcrypt", when="+crypto")
+    conflicts("+crypto", when="platform=windows")
 
     depends_on("python+shared", when="+python")
     extends("python", when="+python")
 
+    # CMake was added in version 1.1.35
+    conflicts("build_system=cmake", when="@:1.1.34")
+
     def url_for_version(self, v):
         if v > Version("1.1.34"):
             return f"https://download.gnome.org/sources/libxslt/{v.up_to(2)}/libxslt-{v}.tar.xz"
         else:
             return f"http://xmlsoft.org/sources/libxslt-{v}.tar.gz"
 
+    @run_after("install")
+    @on_package_attributes(run_tests=True)
+    def import_module_test(self):
+        if self.spec.satisfies("+python"):
+            with working_dir("spack-test", create=True):
+                python("-c", "import libxslt")
```

Comment:
(Optional) What about this, which would build on the module import testing already present?
```
    @property
    def import_modules(self):
        if self.spec.satisfies("+python"):
            return ["libxslt"]
        return []
```

Reply:
That is defined in the `PythonPackage` class. I don't see how that is hooked up here.

Reply:
Don't we get this from `extends(python)` then?

Reply:
It looks like it is just adding the dep on `python` and `python-venv` and then the `extendees` is really only used in the solver.

But maybe I am missing something?